cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -8,16 +8,55 @@ toc::[]
 {odo-title-full} ({odo-title}) is a fast and easy-to-use CLI tool for creating applications on {product-title}. {odo-title} allows developers to concentrate on creating applications without the need to administrate an {product-title} cluster itself.
 Creating deployment configurations, build configurations, service routes and other {product-title} elements are all automated by {odo-title}.
 
-Existing tools such as `oc` are more operations-focused and require a deep understanding of Kubernetes and OpenShift concepts. {odo-title} abstracts away complex Kubernetes and OpenShift concepts allowing developers to focus on what is most important to them: code.
-
+Existing tools such as `oc` are more operations-focused and require a deep understanding of Kubernetes and {product-title} concepts. {odo-title} abstracts away complex Kubernetes and {product-title} concepts allowing developers to focus on what is most important to them: code.
 
+[id="odo-key-features"]
 == Key features
 
 {odo-title} is designed to be simple and concise with the following key features:
 
 * Simple syntax and design centered around concepts familiar to developers, such as projects, applications, and components.
 * Completely client based. No server is required within the {product-title} cluster for deployment.
-* Official full support of Node.js and Java components.
-* Partial compatibility with many languages and frameworks such as Ruby, Perl, PHP, and Python. Run odo catalog list components to see the list of supported and unsupported components.
+* Official support for Node.js and Java components.
+* Partial compatibility with languages and frameworks such as Ruby, Perl, PHP, and Python. 
 * Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real time.
-* Lists all the available components and services from the {product-title} cluster.
\ No newline at end of file
+* Lists all the available components and services from the {product-title} cluster.
+
+[id="odo-supported-languages-and-images"]
+=== Officially supported languages and corresponding container images
+
+.Supported container images
+[options="header"]
+|====
+|Node.js|Java
+|`centos/nodejs-8-centos7`|`redhat-openjdk-18/openjdk18-openshift`
+|`rhoar-nodejs/nodejs-8`|`openjdk/openjdk-11-rhel8`
+|`rhoar-nodejs/nodejs-10`|`openjdk/openjdk-11-rhel7`
+|`bucharestgold/centos7-s2i-nodejs`|
+|`rhscl/nodejs-8-rhel7`|
+|`rhscl/nodejs-10-rhel7`|
+|====
+
+.Listing available container images
+. Log in to the {product-title} cluster with {odo-title}:
++
+----
+$ odo login -u developer -p developer
+----
+
+. List the languages and corresponding container images available for your cluster with `odo catalog list components`.
```

Comment:
How about: List the supported and unsupported components...
This way we can do away with this sentence on line 62: 'The command lists the components and the languages in two sections - supported and unsupported by Red Hat.'

Reply:
@Preeticp that's not what the command exactly do. It's connected to the images available for that particular cluster, not as a list of all odo components/images.

Reply:
I know, we discussed that. 
I am merely suggesting replacing the 'List the languages' part and adding the supported and unsupported part here itself, like this:  List the supported and unsupported components and corresponding container images available for your cluster with `odo catalog list components`.
This makes the sentence redundant: The command lists the components and the languages in two sections - supported and unsupported by Red Hat. which is actually pretty repetitive.

Reply:
see it now, please, I tried to reflow it. Took some time to wrap my head around it.